test/orange/orangeinp/IntersectRegion.test.cc

```diff
@@ -636,11 +638,11 @@ TEST_F(ExtrudedPolygonTest, skewed)
     static char const* const expected_surfaces[]
         = {"Plane: z=10",
            "Plane: z=15",
-           "Plane: n={0.3152,0.3152,-0.89516}, d=-6.9658",
-           "Plane: n={-0.8165,0.40825,0.40825}, d=3.4701",
-           "Plane: n={0.35448,-0.70895,0.6097}, d=3.6511",
-           "Plane: n={0.45718,0.22859,-0.8595}, d=-5.1204",
            "Plane: n={-0.85138,0.42569,0.3065}, d=0.34055",
```

Comment:
I thought something was wrong here because the senses don't change on any surfaces but it actually correct.

![PXL_20250722_194120876~2](https://github.com/user-attachments/assets/c05b5c5b-9f7d-4909-9d20-954e3c0f479a)